### DISCUSSION TITLE
Using momentuHMM with GLS data

### DISCUSSION THREAD

**Discussion by ghost** [bmcclintock/momentuHMM#70] [Q&A] [answered]
Hello,
All the examples of using momentuHMM I have seen online have used GPS data. Is it possible to use this package in the same way but on geolocator (GLS) data (which can have high error)? I have seabird GLS data, so two location estimates every day and I'd like to try and identify migration and breeding periods. Any help would be really appreciated.
Thank you!

**Comment by bmcclintock** [comment, marked as answer]
Movement HMMs have been applied to GLS data (e.g. [Patterson et al. 2009](https://doi.org/10.1111/j.1365-2656.2009.01583.x), [Whoriskey et al. 2017](https://doi.org/10.1002/ece3.2795)). So while I cannot say whether or not this is appropriate for your particular data set, momentuHMM can be used to fit HMMs to GLS data (or any other type of telemetry location data) in the same way. Assuming you have estimates for the location error, multiple imputation would be one way to account for temporal irregularity and/or measurement error.
